bundles/org.openhab.binding.nobohub/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,127 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="nobohub"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="zone">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="nobohub"/>
+		</supported-bridge-type-refs>
+
+		<label>Zone</label>
+		<description>A zone can contain several Nobo devices</description>
+
+		<channels>
+			<channel id="activeWeekProfileName" typeId="activeWeekProfileName-channel-type"/>
+			<channel id="activeWeekProfile" typeId="activeWeekProfile-channel-type"/>
+			<channel id="comfortTemperature" typeId="comfort-temperature-channel-type"/>
+			<channel id="ecoTemperature" typeId="eco-temperature-channel-type"/>
+			<channel id="currentTemperature" typeId="temperature-channel-type"/>
+			<channel id="calculatedWeekProfileStatus" typeId="activeOverrideName-channel-type"/>
+		</channels>
+
+		<properties>
+			<property name="vendor">Glen Dimplex Nobo</property>
+		</properties>
+		<representation-property>name</representation-property>
+
+		<config-description>
+			<parameter name="id" type="integer" required="true">
+				<label>Id</label>
+				<description>Id of the Zone</description>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<thing-type id="component">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="nobohub"/>
+		</supported-bridge-type-refs>
+
+		<label>Component</label>
+		<description>A component is an oven, a switch or a floor termostat</description>
+
+		<channels>
+			<channel id="currentTemperature" typeId="temperature-channel-type"/>
+		</channels>
+
+		<properties>
+			<property name="vendor">Glen Dimplex Nobo</property>
+		</properties>
+		<representation-property>serialNumber</representation-property>
+
+		<config-description>
+			<parameter name="serialNumber" type="text" required="true">
+				<label>Serial Number</label>
+				<description>Serial Number of the component (12 digits)</description>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<channel-type id="activeOverrideName-channel-type">
+		<item-type>String</item-type>
+		<label>Active Override</label>
+		<category>Override</category>
+		<state readOnly="false">
+			<options>
+				<option value="NORMAL">Normal</option>
+				<option value="COMFORT">Comfort</option>
+				<option value="ECO">Eco</option>
+				<option value="Away">Away</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="eco-temperature-channel-type">
+		<item-type>Number</item-type>
+		<label>Eco Temperature</label>
+		<category>Eco Temperature</category>
+		<tags>
+			<tag>TargetTemperature</tag>
```

Comment:
This tag is deprecated, instead use:
```suggestion
			<tag>Setpoint</tag>
			<tag>Temperature</tag>
```

Reply:
Change for Eco/Comfort/Current Temperature